internal/distro/rhel8/distro.go

```diff
@@ -531,10 +532,85 @@ func qemuAssembler(format string, filename string, uefi bool, imageOptions distr
 	return osbuild.NewQEMUAssembler(&options)
 }
 
+func ostreeCommitAssembler(options distro.ImageOptions, arch distro.Arch) *osbuild.Assembler {
+	ref := options.OSTree.Ref
+	if ref == "" {
+		ref = fmt.Sprintf("rhel/8/%s/edge", arch.Name())
+	}
+	return osbuild.NewOSTreeCommitAssembler(
+		&osbuild.OSTreeCommitAssemblerOptions{
+			Ref:    ref,
+			Parent: options.OSTree.Parent,
+			Tar: osbuild.OSTreeCommitAssemblerTarOptions{
+				Filename: "commit.tar",
+			},
+		},
+	)
+}
+
 // New creates a new distro object, defining the supported architectures and image types
 func New() distro.Distro {
 	const GigaByte = 1024 * 1024 * 1024
 
+	edgeImgType := imageType{
+		name:     "rhel-edge-commit",
+		filename: "commit.tar",
+		mimeType: "application/x-tar",
+		packages: []string{
+			"rhel-release", // TODO: is this correct for Edge?
+			"glibc", "glibc-minimal-langpack", "nss-altfiles",
+			"sssd-client", "libsss_sudo", "shadow-utils",
+			"kernel",
+			"dracut-config-generic", "dracut-network",
+			"rpm-ostree", "polkit", "lvm2",
+			"chrony", "zram",
```

Comment:
I don't believe zram is available in rhel. Please remove or comment out for now.